src/BTVNanoCommissioning/helpers/definitions.py

```diff
@@ -753,7 +753,7 @@ def definitions():
             "format_unit": "2f",
             "format_unit_digits": 2,
             "bins": 100,
-            "manual_ranges": [0.0, 0.0],
+            "manual_ranges": [0.0, 0.03],
```

Comment:
Went through this calculation as well, should be at least 0.3 (or even better: 0.4, because we have AK4 jets).

More examples can be found here: https://github.com/AnnikaStein/DeepJet/blob/adversarial_training/calculate_epsilon.ipynb - the values can be interpreted as half of the min-max filled ranges, which is a way to describe the estimated spread of the feature to be used when scaling to unit SD. When you replace `print(scale_epsilon)` e.g. in Cell `In [80]` of the linked notebook with `print(range_inputs, scale_epsilon)`, you can get the raw min/max values as well, not only the distance between them. And these could be more flexible than the rather tight values that are used here in `"manual_ranges"` currently, which I expect to come from a rather small set of inputs of only one process, not a merged file containing QCD&TTbar events.

Note: The files I used for the calculation are stored in `phys_btag` (BTV EOS).

Reply:
Thanks for look into the details, indeed I only go through the ttbar files. I would run through the notebook and update the ranges again.